README.md

```diff
@@ -48,22 +48,21 @@ to address this manual step.
 
 The tooling does not handle all situations and sometimes the generated code will need manual tweeks to get it to compile.
 If this occurs when generating a newer version of an existing package, it can be helpful to regenerate the older version
-to see what changes were made.
-
-The tooling has evolved over time and therefore when generating new packages, you may see edits made to existing packages.
-This is because the new package has a dependency on an existing package, it was regenerated and changes were detected from
-when it was originally generated.
+to see what customizations to the generated code were made.
 
 #### Workflow
 
 * Generate reference package and its depencencies running the `./generate.sh --package <package>,<version>` script.
-* Revert changes for packages that were already existed in the repository.
-* Add `DependencyPackageProjects` for all new projects in the [eng/Build.props](https://github.com/dotnet/source-build-reference-packages/blob/main/eng/Build.props#L9)
+* Revert any changes to packages that already existed in the repository.
```

Comment:
I feel like we should have a test which validates the packages are up-to-date wrt the generator, specifically for GenAPI dependency flow PRs. It's really annoying to have to revert these changes.

Reply:
How often are you seeing changes that aren't intentional customizations now that everything has been regenerated with the genapi based tooling?  Regenerating the code is not feasible at this point for two reasons.

1. The tooling does not support re-generation scenarios today.  There are some tracking issues that prevent this today.
2. There are a number of intentional changes - e.g. explicit package upgrades to address CVEs as well as code modifications that genAPI does not support.

Reply:
I had to revert modifications to at least 2 packages I can remember for my changes in https://github.com/dotnet/source-build-reference-packages/pull/815.

Reply:
Were they necessary changes that were manually applied to the packages to address a CVE or to get the code to compile or were they changes because genAPI was "improved"?

Reply:
I think it was just "improved".

Reply:
If you feel like regenerating the packages on dependency flow would improve the workflow, please log an issue for this.